examples/panel/chat-with-github/README.md

```diff
@@ -0,0 +1,15 @@
+# Chat with Github
+
+This Panel app enables users to chat with their GitHub repositories using the llama_index [GithubRepositoryReader](https://docs.llamaindex.ai/en/stable/examples/data_connectors/GithubRepositoryReaderDemo.html).
+
+To run the app you need to set a couple of [environment variables](https://docs.cloud.ploomber.io/en/latest/user-guide/env-vars.html):
+
+* OPENAI_API_KEY
+* GITHUB_TOKEN
```

Comment:
is the github token mandatory? the github's API rates limits you without a token but you can still use it - so I think this might be optiona

Reply:
I see this in the llama-index code:
```
        if github_token is None:
            github_token = os.getenv("GITHUB_TOKEN")
            if github_token is None:
                raise ValueError(
                    "Please provide a Github token. "
                    "You can do so by passing it as an argument or "
                    + "by setting the GITHUB_TOKEN environment variable."
                )
```

It's also mentioned in the docs: https://docs.llamaindex.ai/en/latest/examples/data_connectors/GithubRepositoryReaderDemo.html